File: src/main/java/gg/mineral/practice/match/OldStyleKnockback.java
package gg.mineral.practice.match;

public class OldStyleKnockback extends CustomKnockback {
    public OldStyleKnockback() {
        super();
        setFriction(2.0D);
        setHorizontal(0.35D);
        setVertical(0.35D);
        setVerticalLimit(0.4D);
        setHorizontalExtra(0.425D);
        setVerticalExtra(0.085D);
    }
}

File: src/main/kotlin/gg/mineral/practice/match/knockback/OldStyleKnockback.kt
package gg.mineral.practice.match.knockback

class OldStyleKnockback : CustomKnockback() {
    init {
        friction = 2.0
        horizontal = 0.35
        vertical = 0.35
        verticalLimit = 0.4
        horizontalExtra = 0.425
        verticalExtra = 0.085
    }
}
